liquid-staking/src/basics/events.rs

```diff
@@ -35,7 +35,7 @@ pub struct RemoveLiquidityEvent<M: ManagedTypeApi> {
 
 #[multiversx_sc::module]
 pub trait EventsModule:
-    crate::config::ConfigModule
+    crate::setup::config::ConfigModule
```

Comment:
Can add some more events:
```
#[event("governance_contract_set")]
fn governance_contract_set(
    #[indexed] new_address: ManagedAddress,
);

#[event("vote_cast")]
fn vote_cast(
    #[indexed] voter: ManagedAddress,
    #[indexed] proposal: ManagedBuffer,
    vote_type: ManagedBuffer,
    amount_locked: BigUint,
    lock_until: Timestamp,
);

#[event("tokens_locked_for_vote")]
fn tokens_locked_for_vote(
    #[indexed] voter: ManagedAddress,
    amount: BigUint,
    lock_until: Timestamp,
);
```

Reply:
governance also makes an indexed event. we probably don't need that.

`tokens_locked_for_vote` looks ok